Source/LocalNetAppChat/LocalNetAppChat.Bot/PluginProcessor/IPlugin.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace LocalNetAppChat.Bot.PluginProcessor
+{
+    public interface IPlugin
+    {
+        string ExecuteCommand(string command);
```

Comment:
The ExecuteCommand should already get a tokenized command so the plugins can easily decide who is responsible.